frontend/src/app/supportAdmin/UnarchivePatients/UnarchivePatient.test.tsx

```diff
@@ -613,8 +613,9 @@ const searchByOrgAndFacility = async () => {
   );
   await selectDropdown("Testing facility *", mockFacility1.name);
   await clickSearch();
-  expect(mockNavigate).not.toHaveBeenCalled();
```

Comment:
I do not believe this is accurate because the search should actually call navigate.
https://github.com/CDCgov/prime-simplereport/blob/bdb640e3afb34b42c3afbd07772e9e28a4180dcb/frontend/src/app/supportAdmin/UnarchivePatients/UnarchivePatient.tsx#L212-L224